src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -128,8 +128,8 @@ public sealed partial class Constants
         public const Microsoft.VisualBasic.MsgBoxStyle vbMsgBoxRight = Microsoft.VisualBasic.MsgBoxStyle.MsgBoxRight;
         public const Microsoft.VisualBasic.MsgBoxStyle vbMsgBoxRtlReading = Microsoft.VisualBasic.MsgBoxStyle.MsgBoxRtlReading;
         public const Microsoft.VisualBasic.MsgBoxStyle vbMsgBoxSetForeground = Microsoft.VisualBasic.MsgBoxStyle.MsgBoxSetForeground;
-        [System.ObsoleteAttribute("For a carriage return and line feed, use vbCrLf.  For the current platform's newline, use System.Environment.NewLine.")]
```

Comment:
Looks like this is an old bug (even in full framework):
https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.constants.vbnarrow?view=netcore-3.0
https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.constants.vbnarrow?view=netframework-4.8

Reply:
@mairaw should the "obsolete" warning in the topic include the text from the source explaining why it's obsolete? Eg., in this case in the code we have (even in 4.8) `<Global.System.Obsolete("For a carriage return and line feed, use vbCrLf.  For the current platform's newline, use System.Environment.NewLine.")>` but in the topic linked above it just says `Warning This API is now obsolete.`. If the description was there it would have been clear it was on the wrong member.

Reply:
Sorry I missed this before @danmosemsft. We do show the text as well in the C# summary box (having the attributes show up for other languages is a known issue that hopefully will get fixed shortly):

![image](https://user-images.githubusercontent.com/12971179/62494552-aac54100-b788-11e9-8926-ef06ecaf0704.png)

Reply:
And this is only for 3.0 @ahsonkhan

https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.VisualBasic/Constants.xml#L2284

Showing that for all versions is related to the issue we were discussing internally with some folks about the inability for docs to display/hide the attribute based on the version selected. It's an issue that want to fix soon.